src/fifo/FifoBuffer.cpp

```diff
@@ -14,18 +14,14 @@
  * limitations under the License.
  */
 
-#include <stdint.h>
-#include <time.h>
-#include <memory.h>
-#include <cassert>
 #include <algorithm>
+#include <memory.h>
+#include <stdint.h>
```

Comment:
We might want to use `<cstdint>` instead of `<stdint.h>` as this seems to be the RightWay™. Source: https://stackoverflow.com/questions/13642827/cstdint-vs-stdint-h

Reply:
Hmm. That blog post says that using just <cstdint> is a "bad idea" and that we need to also add:

using std::int32_t;
using std::uint32_t;
etc.

I think that might be a good thing to do throughout Oboe.
Let's maybe do that in a separate CL.